lib/spack/spack/compilers/pgi.py

```diff
@@ -43,6 +44,16 @@ class Pgi(Compiler):
                    'f77' : 'pgi/pgfortran',
                    'fc'  : 'pgi/pgfortran' }
 
+    @property
+    def openmp_flag(self):
+        return "-mp"
+
+    @property
+    def cxx11_flag(self):
+        tty.die("cxx11_flag() is not implemented for pgi. Consider creating a pull-request.")
+        return "-std=c++11"
```

Comment:
From the man page for pgc++, it appears that PGI accepts either `-std=c++11` or `--c++11` so you can remove this message.